grails-app/views/goodsReceiptNote/_header.gsp

```diff
@@ -7,7 +7,9 @@
         </td>
         <td>
             <div class="header">
-                <div style="font-size: 25px; margin-bottom: 10px; font-weight: bold;">${shipment?.status}</div>
+                <div style="font-size: 25px; margin-bottom: 10px; font-weight: bold;">
+                    <g:message code="enum.ShipmentStatusCode.${shipment?.status}" default="${shipment?.status}" />
```

Comment:
@jmiranda yeah, I'm aware that it works because of this:

> So this only works because we wired the toString()

but I also agree that it might be explicit, so that we are protected from any `toString()` refactors. 
I will change that to `shipment.status.code.name`